package.json

```diff
@@ -28,14 +28,17 @@
     "@redhat-cloud-services/frontend-components": "3.2.6",
     "@redhat-cloud-services/frontend-components-notifications": "3.2.2",
     "@redhat-cloud-services/frontend-components-utilities": "3.2.2",
+    "axios": "^0.21.1",
```

Comment:
locking down the dep version here is recommended

Reply:
@AllenBW, what's the reason for locking it? Just out of curiosity

Reply:
so we know the exact version to expect everywhere